exporter/prometheusremotewriteexporter/config.go

```diff
@@ -31,6 +31,8 @@ type Config struct {
 	// See: https://prometheus.io/docs/practices/naming/#metric-names
 	Namespace string `mapstructure:"namespace"`
 
+	WALConfig *walConfig `mapstructure:"wal"`
```

Comment:
Please explain how this interacts with "sending_queue". During the SIG meeting we talked about this as a replacement for sending queue IIRC